clusterloader2/GETTING_STARTED.md

```diff
@@ -0,0 +1,239 @@
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+# At the end we want to collect pod startup latency.
+- name: Collecting pod startup latency
+  measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: gather
+```
+Now, we need to prepare template for our deployment to be created. All values in brackets will be filled from template parameters:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: {{.Group}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    matchLabels:
+      name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        name: {{.Name}}
+        group: {{.Group}}
+    spec:
+      containers:
+      - image: k8s.gcr.io/pause:3.1
+        imagePullPolicy: IfNotPresent
+        name: {{.Name}}
+        resources:
+          requests:
+            cpu: {{.CpuRequest}}
+            memory: {{.MemoryRequest}}
+```
+## Execute test
+Before running test, make sure that kubeconfig current context points to kind cluster:
+```bash
+kubectl config current-context
+kind-test-cluster
+```
+To execute test, run:
+```bash
+cd clusterloader2/
+go run cmd/clusterloader.go --testconfig=config.yaml --nodes=1 --provider=kind --kubeconfig=${HOME}/.kube/config --masterip=127.0.0.1 --mastername=test-cluster --master-internal-ip=127.0.0.1
+```
+
+At the end of clusterloader output you should see pod startup latency and information that test succeeded:
+```
+PodStartupLatency: perc50: 709.983961ms, perc90: 709.983961ms, perc99: 709.983961ms
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s; threshold 2m0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 2s, perc90: 2s, perc99: 2s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s
+PodStartupLatency: perc50: 0s, perc90: 0s, perc99: 0s; threshold 2m0s
+--------------------------------------------------------------------------------
+Test Finished
+Test: ./config.yaml
+Status: Success
+--------------------------------------------------------------------------------
+```
+
+If you want to see when each step of your test is executed, you can add `--v=2` argument.
+
+## Delete kind cluster
+To delete kind cluster, run:
+```bash
+kind delete cluster --name test-cluster
+```
+
+## Running 100-node scale test on GKE
+
+Here you can find general purpose [Load test]. It can be used to test clusters starting from 100 nodes up to 5k nodes.
+
+First, we need to create 100-node cluster.
+```bash
+export ZONE=us-central1-c
+export PROJECT=ENTER_YOUR_PROJECT
+export NETWORK_NAME=cluster-network
+export CLUSTER_NAME=scale-cluster
+gcloud config set project $PROJECT
```

Comment:
I'm against this.
It's starting GKE cluster and not everyone has to use such cluster.

Let's just say that you should create your 100-node cluster.